src/components/pages/Navbar/NavbarItems.js

```diff
@@ -0,0 +1,41 @@
+import { useAuth0 } from '@auth0/auth0-react';
+import { useHistory } from 'react-router-dom';
+import './Navbar.less';
+import { Menu, Button } from 'antd';
+
+const NavbarItems = () => {
+  const { loginWithRedirect } = useAuth0();
+  const { logout, isAuthenticated } = useAuth0();
+  const { push } = useHistory();
+
+  const logoutAuth = () => {
+    localStorage.removeItem('AuthToken');
+    logout({ returnTo: window.location.origin });
+  };
+
+  return (
+    <Menu theme="dark" mode="vertical">
+      {!isAuthenticated && (
+        <Button
+          type="primary"
+          onClick={() => loginWithRedirect()}
+          style={{ marginLeft: 'auto' }}
+        >
+          Login
+        </Button>
+      )}
+      {!isAuthenticated && (
```

Comment:
Since both the Login button and Apply button rely on !isAuthenticated, is there a way to combine both of them after the Logical && sign instead of having them separate? Perhaps wrapping them in <></> would work.